modules/repository/commits_test.go

```diff
@@ -68,7 +68,7 @@ func TestPushCommits_ToAPIPayloadCommits(t *testing.T) {
 
 	assert.Equal(t, "27566bd", payloadCommits[1].ID)
 	assert.Equal(t, "good signed commit (with not yet validated email)", payloadCommits[1].Message)
-	assert.Equal(t, "/user2/repo16/commit/27566bd", payloadCommits[1].URL)
```

Comment:
Hmm…
that change sounds dangerous.
Isn't that basically a breaking API change?

Reply:
See line 53, the previous test is wrong. It should be a full url in the reality but not `/user2/repo16`. So that there is no breaking in fact.